opennms-base-assembly/src/main/filtered/etc/vacuumd-configuration.xml

```diff
@@ -416,7 +428,16 @@
            AND statustime &lt; now() - interval '24 hours'
       </statement>
     </action>
-    
+
+    <action name="clearPathOutages" >
+      <statement>
+        UPDATE alarms
+           SET severity = 2, lastautomationtime = ${_ts}
+         WHERE eventuei = 'uei.opennms.org/nodes/pathOutage'
```

Comment:
I don't see how that statement would guarantee that the `nodeUp` alarm occurred after the `pathOutage` alarm.

If I were to take a go at it, I'd try updating the trigger to pull the `lasteventtime` from the `nodeUp` alarm, and then add this clause to the action:
`AND lasteventtime <= ${_lasteventtime_from_node_up}`

This should guarantee that we received a `nodeUp` event, since the `pathOutage` alarm was last updated.

I might be wrong though, getting the timing right in these statements is tricky.